README.md

```diff
@@ -2,70 +2,60 @@
 Repository for creating installable packages for AdoptOpenJDK releases.
 
 The packages are created using:
-1. The Wix Toolset http://wixtoolset.org (Windows only)
-2. [Packages](http://s.sudre.free.fr/Software/Packages/about.html) (Mac OS)
-3. For putting together `.deb` and `rpms` head to this link: [linuxNew subdir readme](https://github.com/adoptium/installer/tree/master/linuxNew#readme)
+1. [Wix Toolset](http://wixtoolset.org) (Windows), detail see [implementation](./wix)
+2. [Packages](http://s.sudre.free.fr/Software/Packages/about.html) (MacOS), detail see (implementation](./pkgbuild)
+3.  Linux installer include deb and rpm based package, detail see [implementation](./linuxNew)

```

Comment:
change to linux

Reply:
this is a hyperlink, if we keep folder "linuxNew" for now and update it to (./linux) it actually creates a broken link. 
better to keep it for now, and update it when we finalize  the renaming PR

Reply:
once https://github.com/adoptium/installer/pull/478 get merged(today hopefully) will do a rebase, these should be updated accordingly

Reply:
OK, I've landed #478 so you can rebase now.

Reply:
thanks!